tools/crm_resource.h

```diff
@@ -17,11 +17,8 @@
 #include <crm/services.h>
 #include <crm/common/xml.h>
 #include <crm/common/mainloop.h>
-#include <crm/common/output_internal.h>
```

Comment:
We've been mostly replacing these with `crm/common/internal.h` in header files. It's up to you -- we can leave it out and rely on the `<crm_internal.h>` include at the top.